diagnostic.md

```diff
@@ -14,7 +17,8 @@ list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
 <!-- your answer starts here -->
-
+$.child('ol');
+$('ol').find();
```

Comment:
Close, but not quite. `$('ol').children()` would work, as would `$('body').find('li');`. Because the ONLY descendants of OL are LIs, it would not be necessary to use `.find` on OL. Does that make sense?